src/index.js

```diff
@@ -605,6 +574,24 @@ export default class Resizable extends React.Component<ResizableProps, State> {
         className={this.props.className}
         {...this.extendsProps}
       >
+        {this.state.isResizing &&
+        <div
+          style={{
+            height: '100%',
+            width: '100%',
+            backgroundColor: 'rgba(0,0,0,0)',
+            cursor: `${this.state.resizeCursor || 'auto'}`,
+            opacity: '0',
+            position: 'fixed',
```

Comment:
[question] Why use `fixed` instead of `absolute`?

Reply:
Since we want this div to cover the entire viewport, it makes sense to use fixed. Fixed position is relative to the viewport, and setting the width and height to 100% will then make it cover the entire screen.